paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -1055,14 +1056,17 @@ def get_mesos_utilization_error(
     log.debug(region_pool_utilization_dict)
     free_pool_resources = region_pool_utilization_dict['free']
     total_pool_resources = region_pool_utilization_dict['total']
-    free_percs = []
+    usage_percs = []
     for free, total in zip(free_pool_resources, total_pool_resources):
         if math.isclose(total, 0):
             continue
-        free_percs.append(float(free) / float(total))
+        current_load = total - free
+        boosted_load = cluster_boost.get_boosted_load(region, pool, current_load)
```

Comment:
kwargs please, it is too easy to get the arguments in the wrong order

Reply:
done

Reply:
it doesn't look done